spring-cloud-stream/src/main/resources/META-INF/spring.factories

```diff
@@ -0,0 +1,2 @@
+org.springframework.boot.autoconfigure.EnableAutoConfiguration:\
```

Comment:
Wondering we should just pull this in via `@EnableBindings`' imports. We just want to have it activated if the annotation is present (but I guess the `@ConditionalOnBean` on the class would have the same effect eventually, though).